docs/docs/addons-perf.md

```diff
@@ -56,6 +60,33 @@ The following methods use the measurements returned by [`Perf.getLastMeasurement
 
 * * *
 
+## Example Usage:
+
+We will take the comments example from old official tutorial. The example code is availabe on [github](https://github.com/dhyey35/react-example-for-performance/blob/master/public/scripts/example.js).
+
+```javascript
+//Using react-with-addons.js in ES5
+var Perf = React.addons.Perf;
+
+var Comment = React.createClass({
+  rawMarkup: function() {
+    var md = new Remarkable();
+    var rawMarkup = md.render(this.props.children.toString());
+    return { __html: rawMarkup };
+  },
+  //code...
+  
+ReactDOM.render(
+  <CommentBox url='/api/comments' pollInterval={2000} />,
+  document.getElementById('content')
+);
+```
+Open the developer console in the browser and type `Perf.start()` than followed by the action you want to monitor eg: submitting a form. Than type `Perf.stop()` in the console and than type `Perf.getLastMeasurements()` to get the measurements .See the Reference below for more methods.
+
+![](/react/img/docs/addons-perf.png)
+
+A **Live Example** of the above code is available on [react-example-for-performance](https://react-example-for-performance.herokuapp.com/)
```

Comment:
It's best to avoid dependencies on new backends, especially a one-off herokuapp instance. Perhaps this could just be a static file on the react website itself.